cli/src/cli.rs

```diff
@@ -99,6 +99,9 @@ pub enum CliCommand {
         seed: String,
         program_id: Pubkey,
     },
+    WaitForMaxStake {
+        max_stake_percent: f32,
+    },
```

Comment:
Can you please A-Z these additions within the cluster-query sections?

Reply:
done